Editor/MergeManagerPrefab.cs

```diff
@@ -52,6 +52,13 @@ public bool TryInitializeMerge(string prefabPath)
             // Instantiate our object in order to view it while merging.
             ourPrefabInstance = PrefabUtility.InstantiatePrefab(ourPrefab) as GameObject;
             
+            // UI Elements need a Canvas to be displayed correctly:
+            if (ourPrefabInstance.GetComponentInChildren<RectTransform>() != null) {
+                GameObject defaultCanvas = new GameObject("Canvas");
```

Comment:
Does this not cause any issues while building the list of GameObjects for the later merge step?

Reply:
Do you refer to the call to `GetAllObjects` method? It seems to be fine, because it's executed on the prefab not on the instance of the prefab 🙂 
In general the `GetPath()` method was the only instance that I found where we iterate through the hierarchy backwards (which is the only case where the Canvas could cause issues), but that method is never called on the instance, it's only called on the `ours` and `their` prefab.

I might have missed something, but I think the canvas does not cause any issues 🤞